includes/Admin/Settings_Screens/Advertise.php

```diff
@@ -0,0 +1,180 @@
+<?php
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @package FacebookCommerce
+ */
+
+namespace SkyVerge\WooCommerce\Facebook\Admin\Settings_Screens;
+
+defined( 'ABSPATH' ) or exit;
+
+use SkyVerge\WooCommerce\Facebook\Admin;
+use SkyVerge\WooCommerce\PluginFramework\v5_9_0;
+
+/**
+ * The Advertise settings screen object.
+ */
+class Advertise extends Admin\Abstract_Settings_Screen {
+
+
+	/** @var string screen ID */
+	const ID = 'advertise';
+
+
+	/**
+	 * Advertise settings constructor.
+	 *
+	 * @since 2.2.0-dev.1
+	 */
+	public function __construct() {
+
+		$this->id    = self::ID;
+		$this->label = __( 'Advertise', 'facebook-for-woocommerce' );
+		$this->title = __( 'Advertise', 'facebook-for-woocommerce' );
+
+		add_action( 'admin_head', [ $this, 'output_scripts' ] );
+
+		add_action( 'admin_enqueue_scripts', [ $this, 'enqueue_assets' ] );
```

Comment:
Maybe it's a good idea to split these 2 lines into their own method, ex: `init_hooks` and call it in the constructor ... to avoid adding logic directly in the constructor

Reply:
ok

Reply:
see https://github.com/facebookincubator/facebook-for-woocommerce/pull/1665/commits/68fa2477e13a5f6e2cee13c5941ca9abbba0ee49